data-plane/src/server/tls/tls_server.rs

```diff
@@ -63,6 +66,9 @@ pub struct WantsCert<S: Listener> {
     tcp_server: S,
 }
 
+#[cfg(feature = "enclave")]
+pub static TRUSTED_CERT: OnceCell<Vec<u8>> = OnceCell::new();
```

Comment:
nitpick on the name of this, should it be something like TRUSTED_PUB_KEY?

Reply:
Yeah fair updated!